src/scribe_data/cli/get.py

```diff
@@ -100,40 +103,56 @@ def get_data(
 
     # MARK: Get All
     if all:
-        if language:
-            language_or_sub_language = language.split(" ")[0]
-            print(f"Updating all data types for language for {language.title()}")
-            query_data(
-                languages=[language_or_sub_language],
-                data_type=None,
-                output_dir=output_dir,
-                overwrite=overwrite,
-            )
-            print(
-                f"Query completed for all data types with specified language for {language.title()}."
-            )
-
-        elif data_type:
-            print(f"Updating all languages for data type: {data_type.capitalize()}")
-            query_data(
-                languages=None,
-                data_type=[data_type],
-                output_dir=output_dir,
-                overwrite=overwrite,
-            )
-            print(
-                f"Query completed for all languages with specified data type for {data_type.capitalize()}."
-            )
+        # Using wikimedia lexeme based dump
 
+        if wiki_dump:
```

Comment:
```suggestion
        if wikidata_dump:
```

Quick suggestion - where variables and functions are only named with something like `wiki_*` or `wikimedia_*`, I would suggest actually specifying them with a name like `wikidata_*`. More so, since I believe that we'll likely be using different types of dumps from Wikimedia projects, so it could be good to specify which dump we mean specifically.